src/clr-angular/utils/drag-and-drop/if-dragged.ts

```diff
@@ -17,9 +17,9 @@ import {ClrDragEventListener} from "./providers/drag-event-listener";
 @Directive({selector: "[clrIfDragged]"})
 export class ClrIfDragged<T> implements OnDestroy {
     private subscriptions: Subscription[] = [];
-    constructor(private template: TemplateRef<any>, private container: ViewContainerRef,
+    constructor(private template: TemplateRef<any>, @Optional() @SkipSelf() private container: ViewContainerRef,
```

Comment:
I remember we discussed that we should add `@SkipSelf` but why do we need to add `@Optional()`?